### DISCUSSION TITLE
Qrtz - Datastore

### DISCUSSION THREAD

**Discussion by Pinox** [quartznet/quartznet#646] [General] [answered]
I can't see any details in docs that Qrtz is working with a cloud datastore like Azure Tables / NoSQL DB.

I would like to run Qrtz as a micro-service in cloud , currently using sql server but I think a bit of an overkill to run separate database in cloud.

Is there any cheap alternative for the database to run Qrtz in the cloud (with some redundency) that's perhaps not in the documentation.

thanks.

**Comment by lahma** [comment, marked as answer]
There are [3rd party integrations available](https://www.nuget.org/packages?q=quartz+jobstore), for example:

* MongoDB
* Redis
* DynamoDB

So you should be able to use these kind of packages (and hope they've been updated to match 3.x API or use 2.x Quartz).

Hope this helps.

**Comment by Pinox** [comment]
@lahma  thank you.
